content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,362 @@
+(posted) invoice to register a credit note. On the invoice there is a button named :guilabel:`Credit
+note`, click this button to be directed to the :guilabel:`Create credit note` form, then complete
+the following information:
+
+- :guilabel:`Reason`: type the reason for the credit note.
+- :guilabel:`Journal`: select the journal that has to be electronic and has the :guilabel:`Use
+  Documents?` option active.
+- :guilabel:`Document Type`: select the credit note document type.
+- :guilabel:`Reversal Date`: type the date.
+
+Customer debit note
+~~~~~~~~~~~~~~~~~~~
+
+The :doc:`Customer debit note <../accounting/customer_invoices/credit_notes>` is an electronic
+document that, when validated, is sent to |DGI| via Uruware. It is necessary to have a validated
+(posted) invoice to register a debit note. On the action within an invoice there is a option named
+:guilabel:`Debit note`, click on this action to be directed to the :guilabel:`Create credit note`
+form, then complete the following information:
+
+- :guilabel:`Reason`: type the reason for the debit note.
+- :guilabel:`Journal`: select the journal that has to be electronic and has the :guilabel:`Use
+  Documents?` option active.
+- :guilabel:`Copy lines`: mark checkbox to copy the invoice lines to the debit note.
+- :guilabel:`Debit note date`: type the date.
+
+.. note::
+   Confirm the invoice and it will be created with an internal reference. To send the document to
+   |DGI| via Uruware, click on :guilabel:`Send and Print` and select the checkbox :guilabel:`Create
+   CFE`. The legal document sequence (number) is brought from Uruware once the document has been
+   processed. Make sure you have |CAEs| available in Uruware.
+
+.. note::
+   The PDF of the validated document is pulled from Uruware following the specification by the
+   Uruguayan government (DGI).
+
+Addendas and disclosures
+========================
+
+*Addendas* and *disclosures* are additional notes and comments added to an electronic document that
+can be mandatory or optional. To create a new addenda, go to :menuselection:`Accounting --> Settings
+--> DGI --> Addendas and disclosures`.
+
+To create an addenda, the following information is required:
+
+- :guilabel:`Name`: Name of the addenda or mandatory disclosure.
+- :guilabel:`Type`: Select the type of remark, this will add it to the specific section in the XML.
+- :guilabel:`Is legend`: Select this box if the text is a mandatory disclosure, leave it blank if it
+  is additional information.
+- :guilabel:`Content`: Add the complete text of the addenda or disclosure.
+
+Leyenda and additional information in product
+---------------------------------------------
+
+To add a leyenda or additional information to the product and XML, it is necessary to add the
+preconfigured addenda and disclosure to the product in the invoice line. The field called
+:guilabel:`Disclosure` will be used to add the leyenda to the product specified in the line.
```

Comment:
Where is this field exactly? 
```suggestion
preconfigured addenda and disclosure to the product in the invoice line. Add the leyenda in the 
:guilabel:`Disclosure` field of the product specified in the line.
```
(not sure since I don't know where this field is, but I would avoid the phrasing "The field called XXX will be used" in any case :slightly_smiling_face: )

Reply:
The field is in the product line of the invoice. I agree with your suggestion @auva-odoo. 

@samueljlieber